mindbender/maya/plugins/validate_project_editorial_info.py

```diff
@@ -0,0 +1,50 @@
+# validate_project_editorial_info.py
+import pyblish.api
+
+
+class ValidateMindbenderProjectEditInfo(pyblish.api.ContextPlugin):
+    """
+    All the info that gets validated
+    has been set by the projects
+    bat files.
+    If you are surtain that the info
+    is incorrect, talk to
+    project Supervisor with haste !
+
+    """
+
+    label = "Validate Project Edit Info"
+    optional = True
+    order = pyblish.api.ValidatorOrder
+    hosts = ["maya"]
+
+    def process(self, context):
+        from maya import cmds
+
+        sceneIn = cmds.playbackOptions(query=True, animationStartTime=True)
+        sceneOut = cmds.playbackOptions(query=True, animationEndTime=True)
```

Comment:
> I thought it better to take animationStartTime because that is the one that we set in the usersetup.py

The userSetup is actually faulty at the moment. If you start Maya, you can see how it sets the endTime, but not the maxTime, which means the playable range is only 1 frame. This is a good example of where this can get confusing.

> So to me that was the natural choise to validate as I can see the animator using the playback for their own work and then forgetting to set playback to the edits In and Out
but the animation start and end should always be set to the shots info.

That's good. Just make sure to run this by animators and see if you can find out what they would naturally expect. Try not to mention how you've done it, try finding out what they would expect. So as to not affect their expectations.